From 6dd6a6da699330081875365a5c0011899d08730f Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Wed, 3 Sep 2025 21:20:45 +0300
Subject: [PATCH 4/4] cmd: tlv_eeprom: avoid redeclaration of 'i' in
 find_tlv_devices()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

'int i' is already declared in the "for (int i" scope, thus the i from
the outer scope is unused, leading to:

cmd/tlv_eeprom.c: In function ‘find_tlv_devices’:
cmd/tlv_eeprom.c:917:13: warning: unused variable ‘i’ [-Wunused-variable]
  917 |         int i;
      |             ^

To be squashed with commit "cmd: tlv_eeprom: support specifying tlv
eeprom in DT alias tlv[0-255]".

Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 cmd/tlv_eeprom.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cmd/tlv_eeprom.c b/cmd/tlv_eeprom.c
index 27eb4e3488af..becd2e4e9e14 100644
--- a/cmd/tlv_eeprom.c
+++ b/cmd/tlv_eeprom.c
@@ -914,7 +914,6 @@ static int find_tlv_devices(struct udevice **tlv_devices_p)
 	int ret;
 	char alias_name[7];
 	int count_dev = 0;
-	int i;
 	ofnode node;
 	struct udevice *dev;
 
-- 
2.43.0

